README.md

```diff
@@ -23,20 +23,29 @@ This README covers only TV-specific features. For more general documentation and
 
 ### Hermes JS support
 
-As of the 0.71 release, Hermes is fully working on both Apple TV and Android TV, and is enabled by default.
+- As of the 0.71 release, Hermes is fully working on both Apple TV and Android TV, and is enabled by default.
 
 ### React Native new architecture (Fabric) support
 
-- _Apple TV_: Modify your app's Podfile to set the `:fabric_enabled` value to `true` in both iOS and tvOS targets. After that, run `pod install` to pick up the additional pods needed for the new architecture. Components that have not been reimplemented in the new architecture will show up as an "unimplemented component".
-- _Android TV_: To enable Fabric, modify `android/gradle.properties` in your app and set `newArchEnabled=true`, then rebuild your app.
+React Native TV 0.76 enables the New Architecture by default. You can read more about it in this blog post from the RN core team: [The New Architecture Is Here](https://reactnative.dev/blog/2024/10/23/the-new-architecture-is-here)
 
-As of the 0.74 release, bridgeless is the default when Fabric is enabled. If you need to use Fabric without bridgeless on Apple TV, you can override the default by adding the method below in `AppDelegate.mm`:
+If, for any reason, the New Architecture is not behaving properly in your application, there is always the option to opt-out from it until you are ready to turn it on again.
 
-```objc
-- (BOOL)bridgelessEnabled
-{
-  return false;
-}
+To opt-out from the New Architecture:
+
+- _Expo apps_: RN 0.76 will be supported in SDK 52, where you should use the [`newArchEnabled` property of the Expo config](https://docs.expo.dev/versions/unversioned/config/app/#newarchenabled).
```

Comment:
@brentvatne please let me know if this is ok, or if a different wording should be used.